pkg/odo/cli/component/create.go

```diff
@@ -543,8 +543,16 @@ func (co *CreateOptions) Complete(name string, cmd *cobra.Command, args []string
 			}
 		}
 
-		if componentNamespace == "default" {
-			log.Warning("odo may not work as expected in a default project, please run the odo component in a non-default project. To create a new project, use `odo project create`.")
+		// Check whether resource "Project" is supported
+		projectSupported, err := co.Client.IsProjectSupported()
+
+		if err != nil {
+			// we don't need to return (fail) if this happens
```

Comment:
Why are we ignoring the error?

Reply:
My thought was to avoid exiting due to external reasons during validation (i.e., temporary Openshift API unresponsiveness ). Anyhow, I understand this will skip the conditional right below and may cause some troubles if the default namespace is being used. 
I will change it to catch the error and return if you concur.

Reply:
Yeah let's catch and return the error 👍

Reply:
Done :smiley: